### DISCUSSION TITLE
Error when deleting file

### DISCUSSION THREAD

**Discussion by scouech** [directus/directus#5210] [Archive]
When I try to delete a file (Latest version .60) I get this error: 

```
{
  "errors": [
    {
      "message": "E_UNKNOWN: An unknown error happened with the file 54d248b6-7f21-400d-a68d-d31b7c8ae7c5.\n\nError code: undefined\nOriginal stack:\nTypeError: fse.opendir is not a function\n    at LocalFileSystemStorage._flatDirIterator (/var/projects/medito-content-api/node_modules/@directus/drive/dist/LocalFileSystemStorage.js:218:35)\n    at _flatDirIterator.next (<anonymous>)\n    at FilesService.<anonymous> (/var/projects/medito-content-api/node_modules/directus/dist/services/files.js:200:108)\n    at Generator.next (<anonymous>)\n    at fulfilled (/var/projects/medito-content-api/node_modules/directus/dist/services/files.js:5:58)\n    at process._tickCallback (internal/process/next_tick.js:68:7)",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}
```

Any idea? 

Cheers

**Comment by rijkvanzanten** [comment]
Strange 🤔 Is this on a fresh install? Could it be a `node_modules` problem? I haven't seen this on my end / elsewhere

**Reply by scouech** [reply]
mmm not a fresh install but by upgrading to 0.60.. not sure what to do?

**Reply by rijkvanzanten** [reply]
Did you restart the server after the upgrade? That "Is not a function error" is very strange to me.. Feels like an import of a node_module somewhere goes sideways, or that there's a version mismatch somewhere 🤔 That being said, I/others would've gotten the same issue if that was the case

**Reply by scouech** [reply]
you are right! Restarting the sever fixed it! Very odd! But at least now it's working eheh!

Cheers!

**Reply by rijkvanzanten** [reply]
Pfew!
